### PR TITLE
VideoCell/Column implementation

### PR DESCRIPTION
**Pull request by tchainzzz**
Uses `opencv2` to implement a video cell and column. Equipped with temporal cropping/downsampling transformations as well.

### CODE REVIEW

**Comment by krandiash** [comment]
This branch will need to be rebased on the latest changes from `dev`: the PR should be made onto `dev`. @seyuboglu could you peek in and take a look?

File: mosaic/cells/video.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+                break
+            img = F.to_tensor(img)
+            frames.append(img)
+        cap.release()
+        frames = torch.stack(frames, dim=self.time_dim)
+        return frames
+
+    def get(self):
+        with stderr_suppress():
+            frames = self._read_all_frames()  # TODO: support different decoders
```
Comment (seyuboglu) [inline mosaic/cells/video.py:52]: +1 on this TODO

File: mosaic/contrib/video_corruptions/transforms.py [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+            is "anywhere".
+        stack_clips: (Optional; default True): whether to stack clips in a new
+            dimension (used in 3D action recognition backbones), or stack clips by
+            concatenating across the time dimension (used in 2D action recognition
+            backbones). Output shape if True is (n_clips, *video_shape). If False,
+            the output shape has the same number of dimensions as the original
+            video, but the time dimension is extended by a factor of n_clips.
+
+
+    Examples:
```
Comment (seyuboglu) [inline mosaic/contrib/video_corruptions/transforms.py:91]: Awesome documentation!

File: mosaic/contrib/video_corruptions/transforms.py [hunk 3]
```diff
@@ -0,0 +1,192 @@
...
+
+        # Create a VideoCell from "/path/to/video.mp4" with time in dimension one,
+        sampling 10 clips each of length 16, sampling clips equally across the video
+        >>> cell = VideoCell("/path/to/video.mp4",
+            time_dim=1,
+            transform=TemporalCrop(10, 16, time_dim=1)
+            )
+        # output shape: (10, C, 16, H, W)
```
Comment (seyuboglu) [inline mosaic/contrib/video_corruptions/transforms.py:99]: What's C here?

File: mosaic/cells/video.py [hunk 4]
```diff
@@ -0,0 +1,59 @@
+import os
```
Comment (seyuboglu) [inline mosaic/cells/video.py:1]: `.test_video.py.swp` was accidentally committed, we should remove it

File: mosaic/cells/video.py (hunk 1 above)
Comment (seyuboglu) [reply mosaic/cells/video.py:52]: We should save for a future PR though

**Review by seyuboglu — CHANGES_REQUESTED** [review]
Looks great! 
There's a `.swp` file that was accidentally committed and should be removed. Other than that lgtm. 
I agree that we should be able to specify a custom loader, but we can punt to a future PR.

**Pull request merged**